config/relicGrades.json

```diff
@@ -0,0 +1,6 @@
+[
+  { "id": "Bronze", "tier": "Intact" },
```

Comment:
might wanna refactor "tier" to "refinement".

tier also refers to the "age" or Lith/Meso/Neo/Axi quality of relics.